packages/server/test/support/fixtures/projects/e2e/cypress/plugins/index.js

```diff
@@ -25,16 +30,33 @@ module.exports = (on) => {
 
   on('before:browser:launch', (browser, args) => {
     if (browser.family === 'chrome') {
-      // so that screenshot sizes match those of Electron
-      args.push('--window-size=1280,720')
-      // removes stuff from the top of the window to make `window-size` equal the viewport size
-      args.push('--kiosk')
+      cdpPort = _.chain(args)
+      .find((arg) => arg.startsWith('--remote-debugging-port='))
+      .split('=')
+      .last()
+      .toNumber()
+      .value()
     }
-
-    return args
   })
 
   on('task', {
+    'set:device:metrics' () {
+      if (cdpPort) {
```

Comment:
should specifically throw here if `cdpPort` should be defined and this is chrome - this guards against a situation where something else changes the logic and then this falls through because `cdpPort` is not set as we expect to

Reply:
33228d241